inc/html.class.php

```diff
@@ -5102,6 +5102,9 @@ static function fileForRichText($options=array()) {
     *    - pasteZone           string   DOM ID of the paste zone
     *    - dropZone            string   DOM ID of the drop zone
     *    - rand                string   already computed rand value
+    *    - display             boolean  display or return the generated html
+    *
+    * @return mixed          the html if display paremeter is false or true
```

Comment:
this was my first thought, but I looked an other method with the same feature i nthis class, and found something different. I preferred to follow its example to ensure consistency.

By the way the typo exists in the source of my copy paste :/ ; see textarea()

I'll add a commit to fix both, but before doing that, let's decide if we should use mixed or string|void. The later seems better to me.